openai/wandb_logger.py

```diff
@@ -52,13 +56,19 @@ def sync(
             return
 
         if id:
-            fine_tune = FineTune.retrieve(id=id)
+            print("Retrieving fine-tune job...")
+            if legacy:
+                fine_tune = FineTune.retrieve(id=id)
+            else:
+                fine_tune = FineTuningJob.retrieve(id=id)
             fine_tune.pop("events", None)
             fine_tunes = [fine_tune]
-
         else:
             # get list of fine_tune to log
-            fine_tunes = FineTune.list()
+            if legacy:
+                fine_tunes = FineTune.list()
+            else:
+                fine_tunes = FineTuningJob.list()
```

Comment:
Just FYI this will run into our pagination scheme and only return N most recent results (20 by default iirc). You might want to do something like 
```python
fine_tunes = list(FinueTuningJob.auto_paging_iter())
```

To ensure you get all jobs